components/automate-platform-tools/cmd/pg-helper/main.go

```diff
@@ -270,12 +264,6 @@ func newCmd() *cobra.Command {
 		Args:  cobra.ExactArgs(2),
 	}
 
-	sqitchDeployCmd.Flags().StringVarP(
-		&opts.Sqitch.User,
-		"user", "i", "automate",
-		"User to connect with when running sqitch",
-	)
```

Comment:
As far as I can tell, this feature would not have actually worked but we didn't notice it because we always call this with the default argument.  To avoid confusion, I've removed this option and made it an error on the API side to pass anything but the default user "automate".